src/components/form-text-field/form-text-field.tsx

```diff
@@ -0,0 +1,22 @@
+import { useFormContext } from "react-hook-form";
+import { FormInputProps } from "./form-text-field.types.ts";
+import { Input } from "antd";
+
+const FormTextField = ({ name, ...props }: FormInputProps) => {
+    const {
+        register,
+        formState: { errors },
+    } = useFormContext();
+
+    const errorMessage = errors[name]?.message;
+    const helperText = typeof errorMessage === "string" ? errorMessage : "";
+
+    return (
+        <div>
+            <Input {...register(name)} {...props} status={helperText ? "error" : ""} />
+            {helperText && <div style={{ color: "red", marginTop: "5px" }}>{helperText}</div>}
```

Comment:
I also adjusted the form component and substitute the basic JS form with AntDesign Form component, so it could take all of the Ant Form props and styles instead of basic component props for form. Is it OK? We are still able to pass className as a prop along with all of the ant parameters, it may will be useful